bundles/org.openhab.binding.freebox/src/main/java/org/openhab/binding/freebox/internal/api/model/VirtualMachinesResponse.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.freebox.internal.api.model;
+
+import java.util.List;
+
+import org.openhab.binding.freebox.internal.api.RelativePath;
+
+/**
+ * The {@link VirtualMachinesResponse} is the Java class used to map the
+ * response of Lan Hosts Browser API
+ * https://dev.freebox.fr/sdk/os/lan/#lan-browser
+ *
+ * @author Gaël L'hopital - Initial contribution
+ */
+@RelativePath(relativeUrl = "vm/", retryAuth = true)
```

Comment:
You have `VirtualMachineResponse` and `VirtualMachinesResponse` with the same path "vm/", is it normal ?